From b93bc6c5eef8922206859e2958bc8f5743dd8302 Mon Sep 17 00:00:00 2001
From: "Liu, Kai1" <kai1.liu@intel.com>
Date: Fri, 19 Jul 2019 14:44:45 +0800
Subject: [PATCH] [workaround]Disable CFI for omx aac decoder to pass cts

This is a wrokaround patch to disable cfi for omx aac decoder in order to
pass cts android.media.cts.DecoderTestAacDrc#testDecodeUsacLoudnessM4a

Change-Id: I73c8288c492f43ce1b2cd887961948b90031c8b9
Tracked-On:
Signed-off-by: Liu, Kai1 <kai1.liu@intel.com>
---
 media/libstagefright/codecs/aacdec/Android.bp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/media/libstagefright/codecs/aacdec/Android.bp b/media/libstagefright/codecs/aacdec/Android.bp
index 7352854de..70707d457 100644
--- a/media/libstagefright/codecs/aacdec/Android.bp
+++ b/media/libstagefright/codecs/aacdec/Android.bp
@@ -24,9 +24,9 @@ cc_library_shared {
             "signed-integer-overflow",
             "unsigned-integer-overflow",
         ],
-        cfi: true,
+        cfi: false,
         diag: {
-            cfi: true,
+            cfi: false,
         },
     },
 
-- 
2.21.0

